en/dev_setup/dev_env_linux_ubuntu.md

```diff
@@ -207,15 +206,6 @@ To install the development toolchain:
 :::
 
 
-<a id="fast_dds"></a>
```

Comment:
IF we have an installation topic(s), this would be added back or replaced with link to new information.